invio/docker-compose.yml

```
@@ -0,0 +1,33 @@
+services:
+
+  app_proxy:
+    environment:
+      APP_HOST: invio_frontend_1
+      APP_PORT: 8000
+      PROXY_AUTH_ADD: 'false'
+
+  backend:
+    image: ghcr.io/kittendevv/invio-backend:v1.7.0@sha256:1d7bc45a698e6ee753e34bbb8e7d1d56c8afc338a6cca8f025894667130555ed
+    environment:
+      ADMIN_USER: ${ADMIN_USER:-admin}
+      ADMIN_PASS: ${ADMIN_PASS:-supersecret}
+      JWT_SECRET: ${APP_SEED}
+      SESSION_TTL_SECONDS: ${SESSION_TTL_SECONDS:-43200}
+      SECURE_HEADERS_DISABLED: ${SECURE_HEADERS_DISABLED:-false}
+      ENABLE_HSTS: ${ENABLE_HSTS:-true}
+      DATABASE_PATH: ${DATABASE_PATH:-/app/data/invio.db}
+    volumes:
+      - ${APP_DATA_DIR}/data:/app/data
```

We can store app data in a separate directory. So now  you need to create a new directory inside data directory.
```
invio
├── data
│   ├── app
│   │   └── .gitkeep
│   └── db
│       └── .gitkeep
├── docker-compose.yml
└── umbrel-app.yml

4 directories, 4 files
```
```suggestion
      - ${APP_DATA_DIR}/data/app:/app/data
```